### PR TITLE
Split OTT Messengers into individual pages

### PR DESCRIPTION
**Pull request by alavers**
screen shot 2017-01-18 at 1 17 51 pm

### CODE REVIEW

File: pages/guide/facebook-messenger.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+# Facebook Messenger
+
+Our Facebook Messenger integration allows your users to send you Facebook messages through your Facebook Page which you receive in a business system or with Webhooks. Your replies are sent back as a Facebook message to the user.
+
+### Configuring Facebook Messenger
+
+To use the [Facebook Messenger integration](https://app.smooch.io/integrations/messenger), you need a Facebook account with permissions to manage a published Facebook page. You then need to visit the Facebook Messenger integration page and press on "Connect your Facebook account". This will redirect you to Facebook where you will need to allow Smooch to access your account information.
+
+Once your Facebook account is connected to Smooch, select the Facebook page you want to use and hit save. You should now be able to receive messages from your Facebook page and reply via any of your configured channels.
```
Comment (jpjoyal) [inline pages/guide/facebook-messenger.md]: suggest "reply via any of your configured channels" -> "reply via any of your configured business system or with the Smooch API"

File: pages/guide/facebook-messenger.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+![Facebook Messenger Integration Page Settings](/images/messenger_settings.png)
+
+
+### Action Buttons
+
+Action buttons work as expected. A `link` action button will bring the user to the specified URL while a `postback` action button will trigger any [webhook](https://docs.smooch.io/rest/#webhooks) listening to the `postback` trigger.
+
+Because of new [Facebook terms of service](https://developers.facebook.com/policy/#messengerplatform), payment requests (action buttons of type `buy`) cannot currently be sent to Messenger.
```
Comment (jpjoyal) [inline pages/guide/facebook-messenger.md]: cue for @dannytranlx to doc his new payment support here once ready

File: pages/guide/line.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+layout: two-column
+---
+
+# LINE
+
+LINE is a Japan based rapidly growing mobile messaging app that offers free voice and video calls, messages, and a limitless variety of stickers. With over 600 million users worldwide, LINE's expanding platform is a great way to stay in touch with your customers.
+
+Our LINE integration allows your users to send messages to your LINE bot which you receive in a business system or with Webhooks. Your replies are sent back as coming from your LINE bot.
+
+You can also receive the user messages through a [webhook](https://docs.smooch.io/rest#webhooks).
```
Comment (jpjoyal) [inline pages/guide/line.md]: just link webhooks above instead of having this repetitive line

File: pages/guide/facebook-messenger.md (hunk 2 above)
Comment (dannytranlx) [inline pages/guide/facebook-messenger.md]: 👋

File: pages/guide/telegram.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+<aside class="notice">
+Bots cannot retrieve files bigger than 20MB. If a user attempts to send a file bigger than the limit, you will receive an alert.
+</aside>
+
+When you reply from any of the configured backend channels, Telegram users will receive the message as though it was coming from the bot.
+
+### Action Buttons
+
+When used with Telegram, action buttons work as expected. When a button with type of `link` will bring the user to a web page, while postback buttons will trigger any Smooch webhook listening to the `postback` trigger.
```
Comment (jpjoyal) [inline pages/guide/telegram.md]: "When a button with..." -> "A button with"

File: pages/guide/telegram.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+After choosing the bot's username, BotFather will generate an access token for the API, which Smooch will need to integrate with Telegram.
+
+### Configuring Telegram
+
+![Telegram Integration Page Settings](/images/telegram_not_integrated.png)
+
+Once you have created the bot, all you have to do is enter the bot's access token on the [integration page](https://app.smooch.io/integrations/telegram). Your bot should now relay any message it receives to your backend channel.
+
+### Conversations Through the Bot
+To send a user to the bot, Telegram users have to search for the username of your bot (obtained while creating a bot with BotFather). Once they have added your bot to their contacts, your bot will act as any other Telegram user. Any user will be able to send normal text messages to your bot or any of the supported file formats (documents, images, sound). If the user sends a file, the message will contain a link to the file that was sent.
```
Comment (jpjoyal) [inline pages/guide/telegram.md]: Add a mention for linking users to the bot with telegram.me/<bot_username>

File: pages/guide/telegram.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+![Create a bot with BotFather](/images/botfather_create_bot.png)
+
+After choosing the bot's username, BotFather will generate an access token for the API, which Smooch will need to integrate with Telegram.
+
+### Configuring Telegram
+
+![Telegram Integration Page Settings](/images/telegram_not_integrated.png)
+
+Once you have created the bot, all you have to do is enter the bot's access token on the [integration page](https://app.smooch.io/integrations/telegram). Your bot should now relay any message it receives to your backend channel.
```
Comment (jpjoyal) [inline pages/guide/telegram.md]: suggest: "Your bot should now relay any message it receives to your backend channel." -> "Smooch will now relay any message it receives to your business systems"

Comment (alavers) [inline pages/guide/telegram.md]: screen shot 2017-01-18 at 2 10 53 pm


https://i.imgur.com/em8E3DB.gif?stap

**Pull request merged**